content/es/community/static/cncf-code-of-conduct.md

```diff
@@ -0,0 +1,29 @@
+Código de Conducta para la Comunidad de la CNCF v1.0
+----------------------------------------------------
+
+### Código de Conducta para los Colaboradores
+
+Como colaboradores y mantenedores de este proyecto, y en el interés de promover una comunidad abierta y cordial, nos comprometemos a respetar a todas las personas que colaboren denunciando problemas, publicando solicitudes de función, actualizando documentación, presentando pedidos de validación o parches, entre otras actividades.
+
+Estamos comprometidos con hacer de la participación de todos los involucrados en este proyecto una experiencia libre de hostigamiento de cualquier índole, sin importar su nivel de experiencia, género, identidad y expresión de género, orientación sexual, discapacidad, aspecto personal, tamaño corporal, raza, grupo étnico, edad, religión ni su nacionalidad.
+
+Algunos ejemplos de conducta inaceptable imputable a los participantes pueden ser:
+
+-	El uso de palabras o imágenes sesgadas de contenido sexual.
+-	Ataques personales
+-	Comentarios provocadores, insultantes o despectivos
+-	Hostigamiento público o privado
+-	Publicar información ajena privada, como ser domicilios particulares o electrónicos, sin contar con permiso explícito para hacerlo
```

Comment:
s/como ser domicilios particulares o electrónicos/como ser domicilios particulares o cuentas electrónicas

Reply:
Update requested in https://github.com/cncf/foundation/pull/35

Reply:
Fixed in https://github.com/kubernetes/website/commit/21c69932e27b921cb657a09d7e7ff17e2a8938f0